packages/ozone/src/db/migrations/20240710T051104136Z-sets.ts

```diff
@@ -0,0 +1,63 @@
+import { Kysely, sql } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  // Create the sets table
+  await db.schema
+    .createTable('ozone_set')
+    .addColumn('id', 'serial', (col) => col.primaryKey())
+    .addColumn('name', 'varchar(128)', (col) => col.notNull().unique())
+    .addColumn('description', 'varchar(1024)')
+    .addColumn('createdAt', 'timestamptz', (col) =>
+      col.defaultTo(sql`now()`).notNull(),
+    )
+    .addColumn('updatedAt', 'timestamptz', (col) =>
+      col.defaultTo(sql`now()`).notNull(),
+    )
+    .execute()
+
+  // Create the set values table
+  await db.schema
+    .createTable('ozone_set_value')
+    .addColumn('id', 'bigserial', (col) => col.primaryKey())
+    .addColumn('setId', 'integer', (col) =>
+      col.notNull().references('ozone_set.id').onDelete('cascade'),
+    )
+    .addColumn('value', 'varchar(255)', (col) => col.notNull())
+    .addColumn('createdAt', 'timestamptz', (col) =>
+      col.defaultTo(sql`now()`).notNull(),
+    )
+    .execute()
+
+  // Add indexes for better performance
```

Comment:
These indexes don't especially cater to some of the pagination that we do, and some small tweaks may make a big difference.  For example we have two separate indexes for `ozone_set_value` on `setId` and `value`.  The access pattern we have for paginating over values within a set is to filter by `setId` and order those by `value`.  To best support this pagination we probably want a single index on `(setId, value)` rather than two independent indexes on each column.

Reply:
Ah I see that we do have a unique constraint on `(setId, value)` which would do the trick there.  So perhaps we can instead get rid of some of these indexes! 🎉

Reply:
yep yep. leaving the index on `name`